whileLoop/aufgabe2.java

```diff
@@ -0,0 +1,29 @@
+package whileLoop;
+/*
+* Ergänzen Sie folgende Funktion, so dass die Summe der im Array arr enthaltenen
+* Zahlen berechnet wird. Verwenden Sie dafür eine While Schleife.
+* public static double array_summe(double[] arr) {
+* TODO: Summe implementieren
+* }
+* Überprüfen Sie Ihre eigene Implementierung anhand von Aufrufen in einer main
+* Methode.
+ */
+public class aufgabe2 {
```

Comment:
Class name shoul'd start from the upper case and no numbers in it, e.g. `AufgabeZwei`